packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5964,6 +5968,23 @@ export default class Meeting extends StatelessWebexPlugin {
     return !this.isRoapInProgress;
   }
 
+  private forceSendStatsReport = async ({callFrom}: {callFrom?: string}) => {
```

Comment:
nitpicking: for me, `force` sounds a bit strong here, I think `sendStatsReportNow` would be more accurate, but you can leave it as it is

Reply:
I left it the same since it's force in media-core and wcme, and this was the verbiage marcin and I discussed.